### DISCUSSION TITLE
Feature request: add the ability to play other official songs

### DISCUSSION THREAD

**Discussion by sauloco** [sircharlo/meeting-media-manager#182] [Ideas] [answered]
Sometimes we use songs from [the original songs list](https://www.jw.org/es/biblioteca/musica-canciones/canciones-originales/) or even from [Become Jehovah's Friend section](https://www.jw.org/es/biblioteca/musica-canciones/Hazte-amigo-de-Jehov%C3%A1-canciones/) (links in spanish sorry about that)

Can we have a button to play random songs from these lists too?

I might help to implement it, obviously it's a bit harder that the other random button because it needs to be aware of the language.

**Comment by sircharlo** [comment, marked as answer]
I've thought about this; here's what gave me pause. The direction provided to elders clearly states that **only** Kingdom songs from the _“Sing Out Joyfully”​—Meetings_ series should be played before and after congregation meetings, and not other series such as the original songs.

I'm not sure that adding the possibility of using different song series would be beneficial for JWMMF's main intended use case. I would like to avoid scenarios where JWMMF makes it easier to forgo published direction.

**Reply by sauloco** [reply]
Thanks for the fast reply, I totally understand.

**Reply by sircharlo** [reply]
Thanks for asking! Don't hesitate if ever you have other ideas or questions.
